src/js/base/Context.js

```diff
@@ -101,7 +101,7 @@ export default class Context {
       return isActivated ? this.layoutInfo.codable.val() : this.layoutInfo.editable.html();
     } else {
       if (isActivated) {
-        this.invoke('codeview.sync', html);
+        this.layoutInfo.codable.val(html);
```

Comment:
Is this line related with this PR?

Reply:
Not sure what's going on there, I didn't do anything withe codeview stuff.